src/modules/mount/mount.conf

```diff
@@ -50,7 +50,13 @@ btrfsSubvolumes:
       subvolume: /@
     - mountPoint: /home
       subvolume: /@home
+    - mountPoint: /root
+      subvolume: /@root
+    - mountPoint: /srv
+      subvolume: /@srv
     - mountPoint: /var/cache
       subvolume: /@cache
     - mountPoint: /var/log
-      subvolume: /@log
\ No newline at end of file
+      subvolume: /@log
+    - mountPoint: /var/tmp
+      subvolume: /@tmp
```

Comment:
This is an example file, not particularly intended for use by actual distro's (if you're a distro, and have specific needs / a style for btrfs subvolumes, you should have a `mount.conf` for your distro). In particular `/srv` seems like a distro-specific thing.